ansible/roles/jupyterhub/templates/docker-compose.yml.j2

```diff
@@ -51,6 +51,17 @@ services:
       - env.service
     command: >
       start-notebook.sh --group=formgrade-{{course_id}} --allow-root
+  setup-course:
+    restart: on-failure
+    image: illumidesk/setup-course
```

Comment:
We are currently using variables to identify the built image and the image placed in the docker-compose yaml, so we need to update the ansible vars/tasks to keep things consistent.